app/src/main/java/org/gnucash/android/ui/transaction/TransactionFormFragment.java

```diff
@@ -139,7 +140,7 @@
 	/**
 	 * Formats a {@link Date} object to time string of format HH:mm e.g. 15:25
 	 */
-	public final static DateFormat TIME_FORMATTER = DateFormat.getTimeInstance();
+	public final static DateFormat TIME_FORMATTER = new SimpleDateFormat("hh:mm a");
```

Comment:
This does not take into account time localization. Not all users have an AM/PM time display format